src/kv/RegionCache.cc

```diff
@@ -26,68 +20,19 @@ RPCContextPtr RegionCache::getRPCContext(Backoffer & bo,
         const auto & meta = region->meta;
         std::vector<metapb::Peer> peers;
         size_t start_index = 0;
```

Comment:
revert this function to comment before https://github.com/tikv/client-c/pull/203